docs/readmes/basics/prerequisites.md

```diff
@@ -15,27 +15,32 @@ Install the following tools:
 1. [Docker](https://www.docker.com) and Docker Compose
 2. [Homebrew](https://brew.sh/) *only* for MacOS users
 3. [VirtualBox](https://www.virtualbox.org/)
-3. [Vagrant](https://vagrantup.com)
+4. [Vagrant](https://vagrantup.com)
 
 Replace `brew` with your OS-appropriate package manager as necessary, or see
 the [pyenv installation instructions](https://github.com/pyenv/pyenv#installation).
 
 ```bash
+brew install go@1.13
 brew install pyenv
 
-# Replace .zshrc with your appropriate shell RC file
+# Replace .zshrc with your appropriate shell RC files
 # IMPORTANT: Use .bash_profile, not .bashrc for bash
+export PATH="/usr/local/opt/go@1.13/bin:$PATH" >> ~/.zshrc
 echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >> ~/.zshrc
```

Comment:
While we're on the subject: can you remove the newlines from this echo cmd -- you'll have to add semicolons instead but it's better for readability as a one-line echo fn.

Be sure to validate this change and the go change
- Remove the relevant commands you currently have in your zshrc
- Run both `echo` commands
- Open a new terminal instance and ensure pyenv/go are working as expected

Reply:
Edit: also add the above^ to your test plan in the PR summary

Reply:
Edit 2: for go: instead of `/usr/local/...` use `$(brew --prefix)/...`. This is standard practice, something to do with configurable Homebrew locations idk